CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ the release.
   ([#1707](https://github.com/open-telemetry/opentelemetry-demo/pull/1707))
 * [chore] Fix gen-proto for accountingservice
   ([#1709](https://github.com/open-telemetry/opentelemetry-demo/pull/1709))
+* [chore] Add depends on to otelcol to wait on healthy opensearch
+  ([#1724](https://github.com/open-telemetry/opentelemetry-demo/pull/1724))
+* [accountingservice] bump OpenTelemetry .NET Automatic Instrumentation
+  to 1.8.0 together with other dependencies
+  ([#1727](https://github.com/open-telemetry/opentelemetry-demo/pull/1727))
+* [chore] Fix binding for host's volume mount
+  ([#1728](https://github.com/open-telemetry/opentelemetry-demo/pull/1728))
```

Comment:
What happened here? 🤔 

The only entry that we should have in the changelog, is the #1725

Reply:
not sure, probably some funky things during merging/formatting. Should be good now, could you double check?